src/widget/tool/removefrienddialog.cpp

```diff
@@ -0,0 +1,30 @@
+#include "removefrienddialog.h"
+#include <QPushButton>
+
+
+RemoveFriendDialog::RemoveFriendDialog(const Friend *f, QWidget *parent)
+    : QDialog(parent), _accepted(false)
+{
+    setWindowFlags(windowFlags() | Qt::Tool);
+    ui.setupUi(this);
+    ui.label->setText(ui.label->text().replace("&lt;name&gt;", f->getDisplayedName()));
+    auto removeButton = ui.buttonBox->button(QDialogButtonBox::Ok);
+    removeButton->setEnabled(false);
+    removeButton->setText(tr("Remove"));
+    connect(ui.yes, SIGNAL(textChanged(QString)), this, SLOT(onTextChanged(QString)));
+    connect(ui.buttonBox, SIGNAL(accepted()), this, SLOT(onAccepted()));
+    connect(ui.buttonBox, SIGNAL(rejected()), this, SLOT(close()));
```

Comment:
Could you change it to new Qt5 SIGNAL/SLOT syntax?
http://wiki.qt.io/New_Signal_Slot_Syntax